sceptre/cli.py

```diff
@@ -630,6 +630,11 @@ def init_project(ctx, project_name):
         folder_path = os.path.join(project_folder, folder)
         os.makedirs(folder_path)
 
+    # Create .gitignore file in project
+    gitignore_path = os.path.join(project_folder, ".gitignore")
+    with open(gitignore_path, "wt") as gitignore_file:
```

Comment:
Why "wt" instead of just "w"?

Reply:
I guess in python text mode is the default behavior and thus "w" would indeed be sufficient.